### PR TITLE
Add default admin users to WCMST1.sql_schema script  

### PR DESCRIPTION
**Pull request by sebinbenjamin**
Adds a default user to the admin panel users so that the WCMST1.sql_schema could be used to speed up the initial setup.

### CODE REVIEW

**Comment by sebinbenjamin** [comment]
Hey, @vishnudas-raveendran  I think there is something wrong with the login. I'm only able to login after reloading the page.
Peek 2019-10-28 09-35

Guess it is a different issue.

**Comment by sebinbenjamin** [comment]
> Hey, @vishnudas-raveendran I think there is something wrong with the login. I'm only able to login after reloading the page.
> Peek 2019-10-28 09-35
> 
> Guess it is a different issue.

This is fixed as of #11  . It was either an issue with file permissions or due to some missing PHP extensions.

**Comment by vishnudas-raveendran** [comment]
Hi @sebinbenjamin , An issue for this #1 exists and a pull request #12 has been submitted. However, we still require following extra additions of this commit you have done in the SQL dump file. 
1) Change of DB name from 'ce'  to wcms.
2) Removing the 'achievements' table from schema and if exists already in db line 167. Use 
`DROP TABLE [IF EXISTS]  [database_name.][schema_name.]table_name;` . Using IF EXISTS will help backward support for those who have already created the DB.
3) Other changes that you have done in .gitignore file and readme of port is accepted in #9  pull request 

You may raise a single new issue for resolving 1 and 2 above. Either you may take it up or I will do it. Ping back based on your availability for resolving it.

**Comment by vishnudas-raveendran** [comment]
May I close the pull request ? @sebinbenjamin

**Comment by sebinbenjamin** [comment]
I missed #1. I think it would be better if you do the other changes.

File: WCMST1.sql_schema [hunk 1]
```diff
@@ -153,8 +153,15 @@ CREATE TABLE IF NOT EXISTS `users` (
...
+--
+INSERT INTO `users`(`username`, `password`) VALUES ('admin','4b99e4ad8eabc68770e4c2bdf32298d8');
 
 
 /*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
 /*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+
+
+-- DROP TABLESPACE `achievements`,`events`,`faculty`,`gallery`,`news`,`notifications`,`papers`,`users`;
```
Comment (sebinbenjamin) [inline WCMST1.sql_schema:167]: A bit confused about what is to be done here. `DROP TABLESPACE` was added to fix a data persistence issue on the MySQL container, which I believe is no longer required as of now.

Is am not sure about the `DROP TABLE [IF EXISTS] [database_name.][schema_name.]table_name;` you suggested.

Comment (sebinbenjamin) [reply WCMST1.sql_schema:167]: I think it would be better for you to fix this in a separate issue/PR.  @vishnudas-raveendran

Comment (vishnudas-raveendran) [reply WCMST1.sql_schema:167]: > Hi @sebinbenjamin , An issue for this #1 exists and a pull request #12 has been submitted. However, we still require following extra additions of this commit you have done in the SQL dump file.
> 
> 1. Change of DB name from 'ce'  to wcms.
> 2. Removing the 'achievements' table from schema and if exists already in db line 167. Use
>    `DROP TABLE [IF EXISTS]  [database_name.][schema_name.]table_name;` . Using IF EXISTS will help backward support for those who have already created the DB.
> 3. Other changes that you have done in .gitignore file and readme of port is accepted in #9  pull request
> 
> You may raise a single new issue for resolving 1 and 2 above. Either you may take it up or I will do it. Ping back based on your availability for resolving it.

Raised Issue #14

**Pull request closed without merging**